core/domain/rights_manager.py

```diff
@@ -633,6 +635,13 @@ def _change_activity_status(
         commit_cmds)
     _update_activity_summary(activity_type, activity_rights)
 
+    if new_status != ACTIVITY_STATUS_PRIVATE:
```

Comment:
Duplication was accidental. This is separate because I only want to update first_published_msec after making sure _save_activity_rights and _update_activity_summary works.